### DISCUSSION TITLE
Codeforce sorusu: iki kere 2037A

### DISCUSSION THREAD

**Discussion by kgNaa1903** [sonsuzus/sonsuzus#499] [Soru Cevap]
Kinich, yeni bir günün başlangıcında uyanır. Telefonunu açar, posta kutusunu kontrol eder ve gizemli bir hediye bulur. Hediyeyi açmaya karar verir.

Kinich, içinde n tamsayısı olan bir dizi a'yı açar. Başlangıçta, Kinich'in puanı 0'dır. Aşağıdaki işlemi herhangi bir sayıda kez yapacaktır:

İki indeks i ve j seçin (1 ≤ i < j ≤ n) öyle ki, ne i ne de j daha önce herhangi bir işlemde seçilmemiş olsun ve ai = aj. O zaman puanına 1 ekler.

Yukarıda bahsedilen işlemi herhangi bir sayıda kez yaptıktan sonra Kinich'in elde edebileceği maksimum puanı bulun.

Girdi:
İlk satırda t (1 ≤ t ≤ 500) — test sayısı.
Her bir test durumu için, ilk satırda n (1 ≤ n ≤ 20) — dizinin uzunluğu.
İkinci satırda n tane tamsayıdan oluşan a dizisi.
Çıktı:
Her bir test durumu için, maksimum elde edilebilecek puanı bir satıra yazdırın.

Örnek:
Girdi:

5
1
1
2
2 2
2
1 2
4
1 2 3 1
6
1 2 3 1 2 3
Çıktı:

0
1
0
1
3
Açıklama:

- ve 3. test vakalarında Kinich herhangi bir işlem yapamaz.

- test vakasında ise Kinich 1. ve 2. index'lerdeki 2'leri seçebilir ve bir puan alır.

- test vakasında, 1. ve 4. index'lerdeki 1'leri seçebilir ve bir puan alır.

`def maksimum_toplam(eleman_sayisi, dizi1, dizi2):
    dizi1.sort(reverse=True)
    dizi2.sort(reverse=True)
    toplam = 0
    for i in range(eleman_sayisi):
        toplam += max(dizi1[i], dizi2[i])
    return toplam

eleman_sayisi = 3
dizi1 = [3, 1, 2]
dizi2 = [2, 5, 4]

print(maksimum_toplam(eleman_sayisi, dizi1, dizi2))`

**Comment by sonsuzus** [comment]
Kodunu cevap olarak ekle. Ayrıca bu haliyle çalışmıyor kodun.
